cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/TicketGrantingTicketImpl.java

```diff
@@ -55,16 +57,22 @@
     private Boolean expired = Boolean.FALSE;
 
     /** Service that produced a proxy-granting ticket. */
-    @Column(name="PROXIED_BY", nullable=true)
+    @Lob
+    @Column(name="PROXIED_BY", nullable=true, length = Integer.MAX_VALUE)
     private Service proxiedBy;
 
     /** The services associated to this ticket. */
     @Lob
-    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = 1000000)
+    @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false, length = Integer.MAX_VALUE)
     private final HashMap<String, Service> services = new HashMap<>();
 
+    /** The PGTs associated to this ticket. */
+    @Lob
```

Comment:
I saw that line, was going to sort this out first since it could have a cascade effect on some of the other persisting logic.